static/src/js/components/Map/ScaleControl.ts

```diff
@@ -19,21 +29,29 @@ class ScaleControl extends Control {
     const { map } = options
 
     // Create the scale control
-    const scaleMetric = new ScaleLine({
+    const scaleMetric = new CustomScaleLine({
       className: `${this.className}__scale--metric ${this.className}__scale`,
       units: 'metric'
     })
     scaleMetric.setMap(map)
 
-    const scaleImperial = new ScaleLine({
+    const scaleImperial = new CustomScaleLine({
       className: `${this.className}__scale--imperial ${this.className}__scale`,
       units: 'imperial'
     })
     scaleImperial.setMap(map)
 
     // Add the scale controls to the element
-    this.element.appendChild(scaleMetric.element)
-    this.element.appendChild(scaleImperial.element)
+    this.element.appendChild(scaleMetric.getElement())
+    this.element.appendChild(scaleImperial.getElement())
+  }
+
+  /**
+   * Get the element for this control
+   * @returns {HTMLElement} The element for this control
+   */
+  getElement() {
```

Comment:
maybe we can push this to an inteface and just implement the method for all these that extend controls